### ISSUE TITLE
OpenApiVersion is not specified in template.yaml

### ISSUE THREAD

**Issue by vt-iwamoto** [aws/aws-sam-cli-app-templates#17]
I think [OpenApiVersion](https://docs.aws.amazon.com/ja_jp/serverless-application-model/latest/developerguide/sam-resource-api.html#sam-api-openapiversion) should be specified in template.yaml to avoid creating an extra `Stage` stage.

**Comment by jfuss** [comment]
This seems like a good improvement. Updating labels.

**Comment by duytrandev** [comment]
hi @jfuss is this still valid since I cannot find any resources of type `AWS::Serverless::Api` in any templates

**Comment by hawflau** [comment]
Hi @duytrandev, for templates with implicit API (i.e. `AWS::Serverless::Function` and `AWS::Serverless::StateMachine` with an `Api` event, you can specify `OpenApiVersion` under `Globals` in the template.

e.g.
```
Globals:
  Api:
    OpenApiVersion: 3.0.1

Resources:
  MyFunction:
    Type: AWS::Serverless::Function
    Properties:
      Events:
        ApiEvent:
           Type: Api
           Properties:
              ...
```

**Comment by duytrandev** [comment]
thanks @hawflau let me try to pick up this ticket and see how it goes

**Comment by qingchm** [comment]
This has been addressed in our most recent release 1.79.0 (https://github.com/aws/aws-sam-cli/releases/tag/v1.79.0). Closing the issue here. Thanks for the ticket!
